Makefile

```diff
@@ -80,6 +80,7 @@ HELM_CHART_LINT_ARGS_$(PACKAGE_NAME) = --set nameOverride='',imagePullSecrets=''
 
 REGISTRY_ORGS ?= xpkg.upbound.io/upbound
 IMAGES = uxp-bootstrapper
+XPKG_REG_ORGS_NO_PROMOTE ?= xpkg.upbound.io/upbound
```

Comment:
@nullable-eth since we aren’t using the xpkg machinery this isn’t going to make a difference right? I think we should just no op in https://github.com/upbound/universal-crossplane/blob/00b078743d122f91dc285b98bd84a66b68ea3d4d/cluster/images/uxp-bootstrapper/Makefile#L35

Reply:
Got it.